### PR TITLE
Fix sticky footer issue

### PR DESCRIPTION
**Pull request by vikas-parashar**
Fixes issue of sticky footer using CSS3 flexbox

Fixes: https://github.com/plone/plonetheme.barceloneta/issues/36

### CODE REVIEW

**Comment by vangheem** [comment]
tests running here: http://jenkins.plone.org/job/pull-request-5.0/792/

**Comment by vangheem** [comment]
Can you add a changelog entry please?

**Comment by davilima6** [comment]
Flexbox is awesome and support is really good (caniuse.com/#search=flexbox) but do we want to introduce it in Barceloneta at this time? I mean, it could have been used for the entire layout but was avoided so I'm not sure we should fix this issue like this. Any comments, @sneridagh @albertcasado ?

**Comment by vikas-parashar** [comment]
@davilima6 Thanks for the feedback. Sticky footer can be achieved by calculating footer height and then by using general CSS method but in our case this isn't possible since users can have different footers. Other method is calculating footer's height using JavaScript and then fixing it by CSS but I thought it will be overdo since this can be achieved by mixing [old and new flexbox](https://css-tricks.com/old-flexbox-and-new-flexbox/) for better browser support.
I am always open to suggestions for achieving this other ways and will be happy to apply them.

**Comment by sneridagh** [comment]
plonetheme.barceloneta is not using Flexbox at all (mainly because it's based on vainilla BS3). Introduce it at this level (thorough the whole page structure) is kind of risky and will require more testing for all viewports. Then we have the cross browser support. I remember deciding for Plone 5 to support from IE9+ onwards. This change will break this, making it IE10+, isn't it?.

It's true that MS have officially deprecated IE10 and backwards, however I would wait for a minor version update (eg. Plone 5.1) to make this change effective.

@vangheem @albertcasado @davilima6  what do you think?

File: plonetheme/barceloneta/theme/less/barceloneta-compiled.css [hunk 1]
```diff
@@ -2270,6 +2274,7 @@ fieldset[disabled] .link-parent.active {
   color: #777777;
 }
 .pagination {
+  display: inline-block;
```
Comment (vangheem) [inline plonetheme/barceloneta/theme/less/barceloneta-compiled.css]: Do we want to switch this to inline-block?

File: plonetheme/barceloneta/theme/less/contents.plone.less [hunk 2]
```diff
@@ -37,9 +44,4 @@ body#visual-portal-wrapper.pat-plone #edit-zone {
 	.contenttype-collection:after {	content: '\e808';}
 	.contenttype-event:after {		content: '\e809';}
 	.contenttype-news-item:after {	content: '\e80f';}
-}
-
-//*// thumbnails for images, leadimages  in listings and portlets
-    .image-icon, .image-tile {
```
Comment (vangheem) [inline plonetheme/barceloneta/theme/less/contents.plone.less]: Why are we removing this as well? Does this rule need a more specific selector maybe?

Comment (vikas-parashar) [inline plonetheme/barceloneta/theme/less/contents.plone.less]: Sorry, I think it was removed by mistake. I've fixed it now.

File: plonetheme/barceloneta/theme/less/barceloneta-compiled.css (hunk 1 above)
Comment (vikas-parashar) [inline plonetheme/barceloneta/theme/less/barceloneta-compiled.css]: It's compiling from [pagination.plone.less, line 4](https://github.com/plone/plonetheme.barceloneta/blob/5633a99349a86a139e28cb58cb0422e357cd4bff/plonetheme/barceloneta/theme/less/pagination.plone.less). I don't know why it wasn't in [barceloneta-compiled.css](https://github.com/plone/plonetheme.barceloneta/blob/5633a99349a86a139e28cb58cb0422e357cd4bff/plonetheme/barceloneta/theme/less/barceloneta-compiled.css) already.

**Comment by jensens** [comment]
There was no activity on this for a while. I close this request for now. If you plan to work further on this please reopen and go on.

**Comment by vikas-parashar** [comment]
Yeah sure

**Pull request closed without merging**